### PR TITLE
fix: Refactor preloads to fix an issue with CDN on EE

### LINKED ISSUE #28359: [Task]: Investigate the delay in making api calls in Appsmith cloud.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

_No response_

### PR DESCRIPTION
**Pull request by SatishGandham**
## Description
Our preloads were not using CDN, this PR fixes it. More details in the linked issue.

#### PR fixes following issue(s)
Fixes #28359
 
 
- Bug fix (non-breaking change which fixes an issue)
 
#### How Has This Been Tested?
- [x] Manual

Should verify CDN once deployed on release.
 
#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/preload-route-chunks.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { getBaseURL } from "@appsmith/utils/preloadHelpers";
```
[the remark this answers is not in the document]
Comment (SatishGandham) [reply app/client/src/preload-route-chunks.ts:1]: Sorry, missed committing that file.

**Review by ankitakinger — APPROVED** [review]
LGTM

**Review by rajatagrawal — APPROVED** [review]

**Pull request merged**